source/fundamentals/builders.txt

```diff
@@ -22,15 +20,10 @@ Builders
 Overview
 --------
 
-This page explains on how to use the available
+This section includes guides on how to use each of the available
 builders, and demonstrates the utility the MongoDB Kotlin driver builder classes
 provide.
 
-.. TODO: when builders pages are added back, rephrase the above paragraph to the following
-.. This section includes guides on how to use each of the available
-.. builders, and demonstrates the utility the MongoDB Kotlin driver builder classes
-.. provide.
-
 The Kotlin driver provides classes to simplify the process for developers
 to use CRUD operations and the Aggregation API. The static utility methods allow you
 to build a query more efficiently.
```

Comment:
S: I can't comment on it directly, but on our time we generally prefer all values to be monospaced -- `"female"` and `29` instead of just quotes. Maybe this change can be pushed to a general cleanup?

Reply:
Updated